### PR TITLE
Add active_status to Facilities API

### PR DESCRIPTION
**Pull request by nfasulo**
## Description of change
This PR resolves /department-of-veterans-affairs/vets-contrib/issues/2785

It adds a flag that shows whether a facility is temporarily closed. For now, only VHA facilities will show `A` for active or `T` for temporarily closed for this field. Other types of facilities will remain `nil` in Ruby or `null` in the database or the json that returns to the end consumer. This is because ArcGIS continues to back non-VHA facilities and does not include this information. 

#### These are the major changes:
- Adds a `active_status` column to the `base_facilities` table through a new database migration
- Changes the FacilityParser and VHAFacility to insert the active_status field into the database
- Changes the FacilitySerializer, GeoSerializer, and CsvSerializer to allow the active_status field to return to the end consumer
- Updates the documentation with language to show this active_status field change

#### Other changes
- Updates existing fixtures with default values for `active_status`.

## Testing done
- Adds a test to `vha_facility_spec.rb` asserting that `active_status` can go into and out of the database correctly
- Adds tests to `facilities_request_spec.rb` asserting that the `active_status` flag returns correctly to end consumers for the various facility types
- Verified that responses come back correctly for json/geojson/csv

#### Unique to this PR
- [x] Make database migration for adding `active_status` column to the base_facility model
- [x] Update serializers for json, geojson, and text/csv to return this column
- [x] Add active_status flag to facilities api

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)

### CODE REVIEW

File: lib/facilities/vha_facility.rb [hunk 1]
```diff
@@ -116,7 +117,7 @@ def mapped_fields_list
         %w[Sta_No NAME CocClassificationID LASTUPDATE Address1 Address2 Address3 MUNICIPALITY STATE
            zip Zip4 Sta_Phone Sta_Fax afterhoursphone Mobile
            patientadvocatephone enrollmentcoordinatorphone pharmacyphone Monday
-           Tuesday Wednesday Thursday Friday Saturday Sunday]
+           Tuesday Wednesday Thursday Friday Saturday Sunday Pod]
```
Comment (edmkitty) [inline lib/facilities/vha_facility.rb]: At first glance, I thought you made up a new day of the week lol

File: modules/va_facilities/README.yml [hunk 2]
```diff
@@ -397,7 +403,7 @@ paths:
...
+                  id,name,station_id,latitude,longitude,facility_type,classification,website,physical_address_1,physical_address_2,physical_address_3,physical_city,physical_state,physical_zip,mailing_address_1,mailing_address_2,mailing_address_3,mailing_city,mailing_state,mailing_zip,phone_main,phone_fax,phone_mental_health_clinic,phone_pharmacy,phone_after_hours,phone_patient_advocate,phone_enrollment_coordinator,hours_monday,hours_tuesay,hours_wednesday,hours_thursday,hours_friday,hours_saturday,hours_sunday,mobile,active_status
                   vc_0101V,Boston Vet Center,0101V,42.3445959000001,-71.0361051099999,vet_center,,,7 Drydock Avenue,Suite 2070,,Boston,MA,2210,,,,,,,857-203-6461 x,,,,,,,800AM-700PM,800AM-800PM,800AM-700PM,800AM-800PM,800AM-600PM,-,-,false
```
Comment (rtravitz) [inline modules/va_facilities/README.yml]: Should these facilities include a value for the `active_status` header added on 406?

Comment (nfasulo) [reply modules/va_facilities/README.yml]: Good catch! Although neither of the current example facilities should have a value for `active_status` or `mobile`. Maybe this is overkill, but I'm gonna add a VHA example facility.

Comment (rtravitz) [reply modules/va_facilities/README.yml]: Not overkill. If we catch it and get the chance to have examples that make sense, then I'm all for taking it!

**Comment by nfasulo** [comment]
Thanks for the heads up, @omgitsbillryan. We are working on a large feature branch that will be merged into `master` next week. Our plan is create a PR to merge our migrations into `master` now so they are in place before this code is merged. Will update this PR once those have been merged into `master`.

File: db/schema.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_09_12_172742) do
+ActiveRecord::Schema.define(version: 2019_08_16_120445) do
```
Comment (omgitsbillryan) [inline db/schema.rb]: Maybe a relic of undoing migrations in this branch? Probably don't need it?

Comment (nfasulo) [reply db/schema.rb]: Yep, now that I have merged in the migration from `master` -> `team-z/facilities-datasource-switchover` -> here, this is gone.

**Pull request merged**